src/torznab.ts

```diff
@@ -807,8 +807,7 @@ async function getAndLogIndexers(
 		);
 		if (!entry) return true;
 		if (entry.firstSearched < skipBefore) return false;
-		if (entry.lastSearched > skipAfter) return false;
-		return true;
+		return entry.lastSearched <= skipAfter;
```

Comment:
The other got reverted as I was reviewing but I think the combined return is less intuitive.

Reply:
Sure, reverted this one too.